src/components/Tabs/Tabs.scss

```diff
@@ -343,6 +342,11 @@ $focus-state-box-shadow-color: rgba(0, 0, 0, 0.8);
   }
 
   .Tab-selected {
+    outline: 3px solid transparent;
```

Comment:
the reason I added this was for Windows High Contrast Mode:

This was BEFORE: 
<img width="341" alt="Capture" src="https://user-images.githubusercontent.com/43223543/100692539-14436880-3359-11eb-97fd-b86766b1dc2b.PNG">

This is what it looks like AFTER:


![{3E68B6E5-DDEE-4D6A-AD7B-01026F4465A3} png](https://user-images.githubusercontent.com/43223543/100693130-38ec1000-335a-11eb-8ea5-8e56b2928ada.jpg)

Reply:
I would just wrap this in a media query so it only applies to high contrast mode. We can also be explicit about the color we use here by including our high contrast color mixin

```suggestion
    @media (-ms-high-contrast: active) {
      outline: 3px solid ms-high-contrast-color('text');
    }
```

`ms-high-contrast-color('text')` this will return the highest contrast value according to whatever the background color is set to. If the background is black, text will always be white etc... so we use this for borders / outlines and such.

Reply:
oooh, I like this 👀 😄

Reply:
We seem to be using this approach, media query + HC mixin, in a lot of places so I am all in favour of using it for consistency. 

That said, it comes with some tradeoffs. The media query only works in Microsoft browsers so a user using WHCM in Chrome would still see the before experience. 

The issue with the transparent outline is that even though the CSS property visually has no impact on the user, it is still rendered by the browser even when its not needed.